src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -10339,6 +10339,11 @@ public abstract class OSPlatformAttribute : System.Attribute
         private protected OSPlatformAttribute(string platformName) { }
         public string PlatformName { get; }
     }
+    [System.AttributeUsageAttribute(System.AttributeTargets.Assembly | System.AttributeTargets.Module | System.AttributeTargets.Class | System.AttributeTargets.Struct | System.AttributeTargets.Enum | System.AttributeTargets.Constructor | System.AttributeTargets.Method | System.AttributeTargets.Property | System.AttributeTargets.Field | System.AttributeTargets.Event, AllowMultiple = true, Inherited = false)]
+    public sealed class RequiresPreviewFeaturesAttribute : System.Attribute
```

Comment:
This does not look like the auto-generated source. You may want to follow https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md to avoid introducing delta with auto-generated source. (If there are any unnecessary delta that sneaked in in the meantime, feel free to include them in this PR as well.)

Reply:
Hmm I think there is something missing here. I don't understand the process here too well though. To update the ref in System.Runtime, the type forward needs to exist in the src? Anyway, to get unblocked here, I create a local file in System.Runtime with this attribute, built it with GenerateReferenceSource to auto-generate the changes for System.Runtime.cs and then deleted the local file. @safern, @ericstj maybe there's a better way? 

Also, is System.Runtime a full facade now?

Reply:
`System.Runtime` has always been problematic to regenerate references for. I think about half the time I try it works and the other half I ping Santi that its not working again 😄

Reply:
> Also, is System.Runtime a full facade now?

It has been like that for a long time already. `GenerateReferenceSource` should work fine if the type is defined in System.Private.CoreLib, make sure to run it with `--no-incremental` when invoking `dotnet build /t:GenerateReferenceSource`, feel free to ping me if you can't get this working.